src/client/java/teammates/client/scripts/helper/DataMigrationForSanitizedDataHelper.java

```diff
@@ -0,0 +1,41 @@
+package teammates.client.scripts.helper;
+
+/**
+ * Utility class that provides methods to help with migrating data to its desanitized form.
+ * Contains a private constructor to prevent instantiation of the class.
```

Comment:
According to [this](http://stackoverflow.com/questions/14398747/hide-utility-class-constructor-utility-classes-should-not-have-a-public-or-def), we should make the class final and add the private constructor. So go ahead. I stand corrected.